ra2ce/graph/vector_network_wrapper.py

```diff
@@ -0,0 +1,321 @@
+import logging
+from pathlib import Path
+from typing import List, Union, Tuple
```

Comment:
```suggestion
from typing import Union
```
You can use both `list` and `tuple` directly since we use Python 3.10